GW2EIEvtcParser/EIData/ProfHelpers/Revenant/ConduitHelper.cs

```diff
@@ -26,16 +26,16 @@ internal static class ConduitHelper
     internal static readonly IReadOnlyList<Buff> Buffs =
     [
         new Buff("Legendary Entity Stance", LegendaryEntityStanceBuff, Source.Conduit, BuffClassification.Other, SkillImages.LegendaryEntityStance),
-        new Buff("Form of the Assassin (1)", FormOfTheAssassin1, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheAssassin),
-        new Buff("Form of the Assassin (2)", FormOfTheAssassin2, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheAssassin),
-        new Buff("Form of the Dervish (1)", FormOfTheDervishBuff1, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheDervish),
-        new Buff("Form of the Dervish (2)", FormOfTheDervishBuff2, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheDervish),
-        new Buff("Form of the Mesmer (1)", FormOfTheMesmer1, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheMesmer),
-        new Buff("Form of the Mesmer (2)", FormOfTheMesmer2, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheMesmer),
-        new Buff("Form of the Monk (1)", FormOfTheMonk1, Source.Conduit, BuffStackType.Queue, 9, BuffClassification.Other, BuffImages.FormOfTheMonk),
-        new Buff("Form of the Monk (2)", FormOfTheMonk2, Source.Conduit, BuffStackType.Queue, 9, BuffClassification.Other, BuffImages.FormOfTheMonk),
-        new Buff("Form of the Warrior (1)", FormOfTheWarrior1, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheWarrior),
-        new Buff("Form of the Warrior (2)", FormOfTheWarrior2, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheWarrior),
+        new Buff("Form of the Dervish (Razah Active)", FormOfTheDervishBuff1, Source.Conduit, BuffClassification.Other, BuffImages.FormOfTheDervish),
```

Comment:
Now that it is known, we should also rename the skill id names for those instead of 1/2, Active/Passive